packages/kit/src/views/FirmwareUpdate/components/FirmwareChangeLogView.tsx

```
@@ -159,8 +158,16 @@ export function FirmwareChangeLogContentView({
         width="100%"
-        type="multiple"
-        defaultValue={defaultExpandedSections}
+        type="single"
+        defaultValue={
+          result?.updateInfos?.firmware?.hasUpgrade
+            ? 'firmware'
+            : result?.updateInfos?.bootloader?.hasUpgrade
+            ? 'bootloader'
+            : result?.updateInfos?.ble?.hasUpgrade
+            ? 'ble'
+            : undefined
```

ESLint

[error] Do not nest ternary expressions.

(no-nested-ternary)